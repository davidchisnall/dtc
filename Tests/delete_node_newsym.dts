/dts-v1/;

/ {
    foo {
    };

    qux: bar {
        foo-prop = "ok";
    };
};

/ {
    /delete-node/ bar;

    quux: bar {
        bar-prop = "ok";
    };

    foo {
        next-quux = <&quux>;
    };
};
