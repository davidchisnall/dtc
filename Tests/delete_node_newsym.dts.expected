/dts-v1/;

/  {

	foo {

		next-quux = <0x1>;
	};
	bar {

		bar-prop = "ok";
		phandle = <0x1>;
	};
};
